test/test_bessel_k.hpp

```diff
@@ -133,7 +133,9 @@ void test_bessel(T, const char* name)
         {{ SC_(-10.0), SC_(1.0), SC_(1.80713289901029454691597861302340015908245782948536080022119e8) }},
         {{ SC_(100.0), SC_(5.0), SC_(7.03986019306167654653386616796116726248616158936088056952477e115) }},
         {{ SC_(100.0), SC_(80.0), SC_(8.39287107246490782848985384895907681748152272748337807033319e-12) }},
-        {{ SC_(-1000.0), SC_(700.0), SC_(6.51561979144735818903553852606383312984409361984128221539405e-31) }},
+        (std::numeric_limits<T>::is_specialized && (std::numeric_limits<T>::min_exponent10 > -300))
```

Comment:
As with the first comment, given that the result is in range, this should work for all T, do you know where things go wrong?

Reply:
Same as above.